lib/utils/telemetry/generatePayload.js

```diff
@@ -108,10 +111,20 @@ module.exports = async (serverless) => {
     return userConfig.get('userId');
   })();
 
+  const isLocallyInstalled = await (async () => {
+    if (serverless) {
+      return serverless.isLocallyInstalled;
+    }
+
+    return Boolean(await resolveLocalServerlessPath());
```

Comment:
This method resolves the path to eventually locally installed `serverless`.  If it exists it doesn't mean we're in context of it. We need to compare it to path of this `serverless` package to be sure (like here: https://github.com/serverless/serverless/blob/e1b9ba46bda28db101f61a53b46a66ab195ad810/lib/cli/handle-error.js#L106)

Reply:
@pgrzesik I just thought that it might be good to introduce an utility `lib/util/is-locally-installed.js` and rely on that.

I also need it in context of interactive CLI refactor :)

Reply:
Right - also good call with introducing a tool. I'll add that

Reply:
On a second though, I'm a little bit uncertain how that should should operate? Should it only involve getting `serverlessPath` and `resolveLocalServerlessPath` and based on comparison return the value? In some places we also rely on `isLocallyInstalled` flag on `serverless` instance, so, do you think it should accept `serverless` instance as param?

Reply:
Util should not rely on `serverless` instance. and here we should rely on `serverless` instance if it exists and otherwise fallback to util

Reply:
I've went with `isUsingLocalInstallation` for `util` name to not clash with `isLocallyInstalled` property from `serverless`